dev-requirements.txt

```diff
@@ -8,3 +8,4 @@ black==22.6.0
 flake8==5.0.4
 mypy==0.971
 isort
+typing_extensions
```

Comment:
The test passes only because it requires installing all dev reqs. In live mode, it would crash because typing_ext is not found.